dashi/poolqc/pooling_qc_sample.py

```diff
@@ -18,70 +20,150 @@
 bcl2fastq = gsiqcetl.bcl2fastq.parse.load_cache(
     gsiqcetl.bcl2fastq.parse.CACHENAME.SAMPLES,
     './data/bcl2fastq_cache.hd5')
+bcl2fastq.rename(columns={'SampleNumberReads': 'Clusters'}, inplace=True)
 bcl2fastq['library'] = bcl2fastq['SampleID'].str.extract('SWID_\d+_(\w+_\d+_.*_\d+_[A-Z]{2})_')
 
 df = bcl2fastq.merge(rnaseq, on='library', how='outer')
 df = df.merge(bamqc, on='library', how='outer')
 df = df.drop(columns=['Sequencer Run Name', 'Lane Number'])
 
-runs = df['Run'].dropna().sort_values(ascending=False).unique()
-
-# Currently using arbitrary values as threshold for visualisation
-INDEX_THRESHOLD = 500000
+runs = df['Run'].sort_values(ascending=False).unique()
+runs = [x for x in runs if str(x) != 'nan']
 
 layout = html.Div(children=[
-    dcc.ConfirmDialog(
-        id='warning',
-        message='The selected run does not return any data. Analysis may have not been completed yet.' '''
+
+    sd.Drawer(id='Filter_drawer',
+              width='40%',
+              docked=False,
+              children=[
+                  html.Div([html.P(children='Report Filters', style={'font-size': '30px', 'padding-top': '10px',
```

Comment:
For consistency with the RNASeq over time report, could you make this [a button](https://github.com/oicr-gsi/dashi/blob/master/dashi/rnaseqc/over_time.py#L165)?

Reply:
Please also look at the conventions used in the RNASeq over time report filter box and match those (no need for a title, no need for "Select A" in each dropdown's title, etc.).